scripts/sql/create_sql_user.py

```diff
@@ -14,42 +14,51 @@
 
 import pyodbc
 import os
+import json
 
 server = os.environ.get("SERVER")
 database = os.environ.get("DATABASE")
 client_id = os.environ.get("CLIENT_ID")
 client_secret = os.environ.get("CLIENT_SECRET")
-login_to_create = os.environ.get("LOGIN_TO_CREATE")
+users_to_create = os.environ.get("USERS_TO_CREATE")
 
 
 def create_con_str(db: str) -> str:
     driver = "{ODBC Driver 18 for SQL Server}"
     return f"DRIVER={driver};SERVER={server};DATABASE={db};ENCRYPT=yes;Authentication=ActiveDirectoryServicePrincipal;UID={client_id};PWD={client_secret}"  # noqa: E501
 
 
+# Map users
+users_data = json.loads(users_to_create)
```

Comment:
nit: I would have the json as a list then you can just

```python
users = json.loads(users_to_create)
```
which is cleaner, imo anyway